### DISCUSSION TITLE
WebView2 Runtime

### DISCUSSION THREAD

**Discussion by iyzerdx** [Alex313031/thorium#609] [General]
Hi, I was using Edge before shifting to Thorium and I uninstalled the Edge and Edge WebView2 Runtime and now there are some apps that may require the latter. So, is there anything available like Thorium that could replace WebView2 Runtime?

**Comment by Alex313031** [comment]
@iyzerdx Not that I am aware of. WebView2 is closed source, and apps that depend on it on Windows can't use anything else, as it is hard coded at compile time.
